pkg/cli/mirror/options.go

```diff
@@ -0,0 +1,44 @@
+package mirror
+
+import (
+	"github.com/RedHatGov/bundle/pkg/cli"
+	imagemanifest "github.com/openshift/oc/pkg/cli/image/manifest"
+	"github.com/spf13/pflag"
+)
+
+type MirrorOptions struct {
+	*cli.RootOptions
+	OutputDir        string
+	ConfigPath       string
+	SkipImagePin     bool
+	ManifestsOnly    bool
+	From             string
+	ToMirror         string
+	DryRun           bool
+	SourceSkipTLS    bool
+	DestSkipTLS      bool
+	SkipVerification bool
+	SkipCleanup      bool
+	FilterOptions    imagemanifest.FilterOptions
+
+	BuildxPlatforms []string
+}
+
+func (o *MirrorOptions) BindFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.ConfigPath, "config", "c", o.ConfigPath, "Path to imageset configuration file")
```

Comment:
Some flag help texts start with upper case letters, some with lower case. Let's choose one (I prefer upper).

Reply:
+1 to upper. I will make this change.